config/localization/en/app.json

```diff
@@ -730,6 +730,29 @@
         "SECONDARY": "ZetaChain is a public, decentralized blockchain and smart contract platform that enables message passing and value transfer between any blockchain."
       }
     },
+    "AFFILIATES": {
+      "AFFILIATES_PROGRAM": "Affiliates Program",
+      "AFFILIATE_LINK": "Affiliate Link",
+      "EARN_FOR_EACH_TRADER": "Earn up to ${AMOUNT_USD}/mo for each new trader",
+      "EARN_FEES": "Earn fees",
+      "REFER_FOR_DISCOUNTS": "Refer a friend and they can receive up to ${AMOUNT_USD} in discounts.",
+      "REFER_FOR_DISCOUNTS_FIRST_ORDER": "Receive up to ${AMOUNT_USD} in fee discounts after placing your first order!",
+      "EARCH_FOR_EACH_TRADER_REFER_FOR_DISCOUNTS": "Earn up to ${AMOUNT_PER_MONTH}/mo for each new trader you refer. Your referral can save up to ${AMOUNT_DISCOUNT} in trading fees. {LEARN_MORE_LINK}",
+      "WANT_TO_VIEW_EARNINGS": "Want to view your earnings? {LINK}",
+      "INVITE_FRIENDS": "Invite Friends",
+      "YOUR_FRIEND": "Your friend",
+      "HAS_REFERRED_YOU": "has referred you to start trading on the world’s most powerful open exchange.",
```

Comment:
Should these be separate? Feels weird to cut translation off mid sentence xD

Reply:
yea haha, the design is a bit difficult to make it in 1 string.
<img width="364" alt="Screenshot 2024-09-04 at 5 15 52 PM" src="https://github.com/user-attachments/assets/2036a111-3c15-408c-9b2d-d5b546c4b644">

Reply:
LOL makes sense